content/posts/July-2021.md

```diff
@@ -37,3 +37,12 @@ is either internal (not user facing) or not finished yet.
   Don't forget to give our GitHub application permissions to use it.
   You can expect more improvements in this field.
   ([packit-service#1167](https://github.com/packit/packit-service/pull/1167))
+
+## Week 30 (July 26th - July 30th)
+
+- Source-git patch metadata: the `squash_commits` key is deprecated.
```

Comment:
if we want to mention this, we should also describe the way it is done currently, you can check Hunor's [doc changes](https://github.com/packit/packit.dev/pull/316/files) about this